docs/interest.md

```diff
@@ -0,0 +1,198 @@
+# WordPress Edge Integrations: Interest
+
+## Function reference
+
+### `get_interest`
+
+Return or alter interest data in the global header.
+
+#### Parameters
+
+__(array)__ Data to pass to the HeaderData class. If an empty arrat is passed, `get_interest` will return all Interest data.
+
+#### Return
+
+_(string)_ The interest data in a JSON encoded string.
+
+#### Example
+
+```php
+use Pantheon\EI\WP\Interest;
+
+// Returns interest data from HeaderData class.
+$interest = Interest\get_interest();
+
+// Manually pass data to interest header.
+$data = ['HTTP_INTEREST' =>'Carl Sagan|Richard Feynman'];
+$interest = Interest\get_interest( $data );
+```
+
+### `set_interest`
+
+Set the interest header key and data.
+
+#### Parameters
+
+$key __(array)__ Key for the header, or array of keys.
```

Comment:
I would wrap variable names in backticks, e.g.:
```suggestion
`$key` __(array)__ Key for the header, or array of keys.
```
🔢

Reply:
Addressed [93e7991](https://github.com/pantheon-systems/edge-integrations-wordpress-sdk/pull/5/commits/93e79914fef2f9bfa99c1eacc9cba00222cfc0c4)